app/views/users/webauthn_setup/new.html.erb

```diff
@@ -33,20 +38,38 @@
 
   <%= hidden_field_tag :platform_authenticator, @platform_authenticator, id: 'platform_authenticator' %>
   <% if !@platform_authenticator %>
-    
-      <%= render ValidatedFieldComponent.new(
-            form: f,
-            name: :name,
-            required: true,
-            label: @presenter.nickname_label,
-            hint: @presenter.device_nickname_hint,
-            input_html: {
-              id: 'nickname',
-              class: 'font-family-mono',
-              size: 16,
-              maxlength: 20,
-            },
-          ) %>
+    <fieldset aria-labelledby="webauthn-intro" class="padding-0 border-0 margin-y-4 margin-x-0">
+      <%= render ProcessListComponent.new(connected: true) do |c| %>
+        <%= c.with_item(heading: t('forms.webauthn_setup.step_1'), heading_id: 'step-1-label') do %>
+          <p><%= t('forms.webauthn_setup.step_1a') %></p>
+          <%= render ValidatedFieldComponent.new(
+                form: f,
+                name: :name,
+                required: true,
+                label: false,
+                hint: @presenter.device_nickname_hint,
+                wrapper_html: { class: 'margin-bottom-0' },
+                input_html: {
+                  aria: { labelledby: 'step-1-label' },
+                  id: 'nickname',
+                  class: 'font-family-mono',
+                  size: 16,
+                  maxlength: 20,
+                },
+              ) %>
+        <% end %>
+        <%= c.with_item(heading: t('forms.webauthn_setup.step_2')) do %>
+            <% if @mobile %>
+              <%= image_tag asset_url('mfa-options/security_key_mobile.gif'), class: 'rounded-xl', alt: t('forms.webauthn_setup.step_2_image_mobile_alt') %>
```

Comment:
I'm surprised this isn't being flagged by [our image size linter](https://github.com/18F/identity-idp/blob/main/lib/linters/image_size_linter.rb), but in any case could we add dimensions to these images to [avoid layout shifts](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/)?

Reply:
> I'm surprised this isn't being flagged by [our image size linter](https://github.com/18F/identity-idp/blob/main/lib/linters/image_size_linter.rb),

[This file is exempted](https://github.com/18F/identity-idp/blob/fc201da4e9ae4dd65bd7a23c52ad6aef03693eb6/.rubocop.yml#L77), which is why the lint didn't raise an issue here. Past @aduth wasn't very helpful in explaining why it was exempted in #7248, but I think the reason is that `@presenter.image_path` producing different images meant it was hard to have a static width and height for that image. But since we're now only showing an image for platform authenticators, I think we can assign a `width` and `height` for the platform authenticator image and un-exempt this file.

Reply:
Removed the exemption and added a fixed width and height. I've set the width and height to the rendered size. The rendered height was an odd size, 193.3px. I've set the height to 193.